.github/workflows/reposchutz.yml

```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: read
+      issues: write
```

Comment:
This *could* also be `pull-requests:`. The [docs](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions) does not make it clear whether this applies to the type of the API or target object (a PR is also an issue in that regard).